src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
It would be good to add :not(:last-of-type) for this margin so the last star doesn't have the margin

Reply:
Done.